bylaws/001-mission-and-structure.md

```diff
@@ -28,6 +28,14 @@ If, in the judgement of the PHP FIG, a Project Representative is acting inapprop
 
 A Project Representative may also simultaneously serve as a Core Committee member, PSR Editor or Working Group member but not as a Secretary.
 
+## Conduct
+
+All individuals involved with FIG either in an official capacity (Secretary, Core Committee, Project Representative, Editor, or Working Group member) or informally (such as participating in discussions) are expected to conduct themselves in a professional manner at all times.  All individuals are expected to behave in a manner consistent with the Code Manifesto.  Failure to do so will be considered grounds for expulsion.
```

Comment:
> Failure to do so will be considered grounds for expulsion.

I'm not sure this statement is necessary. The following paragraphs go on to state what can happen if behaviour is unprofessional but this sentence is very blanket. Expulsion from what? It depends. 'Grounds' is also a word I'd steer clear of as that implies there has to be a reason for expulsion votes, and currently, in bylaws, there isn't.

Reply:
Perhaps also clarify where this is actualised. For example, boiled down this can sound like "People participating in discussions are expected to conduct themselves in a professional manner at all times".

Is anyone professional _at all times_? Is making mistakes (and apologising for them) professional?

Reply:
That's already clarified by the Manifesto itself, point 8: https://github.com/php-fig/fig-standards/pull/1143/files#diff-cf0da61c669e0bb2f7db68b302c98060R17

Reply:
@Jean85 Thanks, I guess it covers the mistakes part. But _at all times_ sounds like I (as a discussion participant) might be excepted to behave professionally even outside of FIG; say in my private life. English is not my first language so I might be off here.

Reply:
from the context and the example, i would expect this to refer to discussions within the fig.

that said, i would hope its possible to expulse somebody for behaviour that is clearly against this manifesto regardless of the context. acting "correct" within the fig but for example spreading misogyny or racism, or harassing people in other contexts should be possible reason for expulsion.

as michelle points out, the process must be clear so the fig does not act on unfounded allegions or rumors.

Reply:
"At all times", at least to a native English speaker, implies "in the context of FIG".  I don't really care how professional you are at your backyard picnic, for instance.

Applying a local CoC to non-local activities is very tricky business.  On the one hand, if someone is openly and publicly racist from the same Twitter account that they talk about FIG stuff, that could be problematic.  On the other, we want to avoid policing people's legal political or personal activities.  That's not a hypothetical concern as, for instance, I still feel that Brendan Eich being forced out at Mozilla was grossly inappropriate.  I'm not sure if there's language we could add there that clarifies a "middle ground" scope effectively.  It would be nice if we could find some verbage for that.